site/pages/api/get-all-templates.api.ts

```diff
@@ -0,0 +1,190 @@
+import { Disruption } from "@create-disruptions-data/shared-ts/disruptionTypes";
+import { Progress, PublishStatus, Severity } from "@create-disruptions-data/shared-ts/enums";
+import {
+    getSortedDisruptionFinalEndDate,
+    sortDisruptionsByStartDate as sortTemplatesByStartDate,
+} from "@create-disruptions-data/shared-ts/utils";
+import { Dayjs } from "dayjs";
+import { NextApiRequest, NextApiResponse } from "next";
+import { VEHICLE_MODES } from "../../constants";
+import { getTemplatesDataFromDynamo } from "../../data/dynamo";
+import { getDisplayByValue, mapValidityPeriods, reduceStringWithEllipsis } from "../../utils";
+import { getSession } from "../../utils/apiUtils/auth";
+import { isLiveDisruption } from "../../utils/dates";
+
+export interface GetDisruptionsApiRequest extends NextApiRequest {
+    body: {
+        orgId: string;
+        start: number;
+    };
+}
+
+export const getTemplateStatus = (disruption: Disruption): Progress => {
+    if (disruption.publishStatus === PublishStatus.draft) {
+        return Progress.draft;
+    }
+
+    if (disruption.publishStatus === PublishStatus.rejected) {
+        return Progress.rejected;
+    }
+
+    if (disruption.publishStatus === PublishStatus.pendingApproval) {
+        return Progress.draftPendingApproval;
+    }
+
+    if (
+        disruption.publishStatus === PublishStatus.editPendingApproval ||
+        disruption.publishStatus === PublishStatus.pendingAndEditing
+    ) {
+        return Progress.editPendingApproval;
```

Comment:
Are these assertions required if we don't have an approval status for templates?

could be simplified to:
```
export const getTemplateStatus = (disruption: Disruption): Progress => {
    if (disruption.publishStatus === PublishStatus.draft) {
        return Progress.draft;
    } else return Progress.open;
};
```

Reply:
done